docs/contributing.rst

```diff
@@ -220,47 +260,19 @@ To run spell checker on Linux box you should install it first:
    $ sudo apt-get install enchant
    $ pip install sphinxcontrib-spelling
 
-Changelog update
```

Comment:
@Dreamsorcerer I think you had Sphinx problems because you didn't keep this title as in examples. I applied a partial revert and there's no errors.